### PR TITLE
UI: added password ui

### PR DESCRIPTION
**Pull request by mgerhardy**
See issue #28

- [x] Integrate new background image

Bildschirmfoto von 2021-04-17 00-13-58

### CODE REVIEW

**Comment by kai-li-wop** [comment]
@mgerhardy Please have a look here #60 (unfinished code), which should provide the new bg picture. Probably I need to add also a shader. I had no chance to test it.

**Comment by mgerhardy** [comment]
I'm done - just let me know if the layout is ok.

**Review by kai-li-wop — APPROVED** [review]
This seems to work simply fine.

File: code/q3_ui/ui_password.c [hunk 1]
```diff
@@ -0,0 +1,158 @@
+/*
+===========================================================================
+Copyright (C) 2009 Poul Sander
+
+This file is part of the Open Arena source code.
```
Comment (robo9k) [inline code/q3_ui/ui_password.c]: I think this is a bit misleading, even if it's still 1:1 the same file. Suggest making this uniform with other WoP code headers and of course keeping Poul and OA in the notice.

File: Makefile [hunk 2]
```diff
@@ -3028,6 +3029,12 @@ endif
 dist:
 	git archive --format zip --output $(CLIENTBIN)-$(VERSION).zip HEAD
 
+%-run: release
```
Comment (robo9k) [inline Makefile]: How is this related to the other code changes?

Comment (mgerhardy) [reply Makefile]: Obviously it is not - should I bring that into `main` outside of the PR?

File: code/q3_ui/ui_password.c (hunk 1 above)
Comment (mgerhardy) [reply code/q3_ui/ui_password.c]: I don't understand in which way I should unify this - I can't make this header to claim that Id Software is the copyright holder of it. It is also not part of the Quake III Arena source code.

Comment (robo9k) [reply code/q3_ui/ui_password.c]: It is part of "World of Padman" now, so that's what the notice should say. As for the new "Copyright (C) 2021 xyz" you can either add your own name or "Padworld Entertainment".
As for uniform code headers I noticed that none of the WoP files have one, must've been wishful thinking on my part.
For new files we're adding I still suggest we put the GPLv2+ header in them, just like id/io Q3

File: Makefile (hunk 2 above)
Comment (robo9k) [reply Makefile]: Yes, please refrain from adding unrelated commits or code changes that are not strictly required for the merge request/issue.
You can leave that one in or just push it to `main` with my approval and without a separate merge request.

Comment (mgerhardy) [reply Makefile]: done

File: code/q3_ui/ui_password.c (hunk 1 above)
Comment (mgerhardy) [reply code/q3_ui/ui_password.c]: I think I'm done.
